packages/eslint-config-expo/utils/extensions.js

```diff
@@ -0,0 +1,23 @@
+const jsExtensions = ['.js', '.jsx'];
+const tsExtensions = ['.ts', '.tsx', '.d.ts'];
+
+const platformSubextensions = ['.android', '.ios', '.web', '.native'];
+
+function computeExpoExtensions(baseExtensions, platformSubextensions) {
+  const expoExtensions = [];
+  for (const expo of ['.expo', '']) {
```

Comment:
expo extensions have been deprecated for years, we shouldn't need to bring it in here.